string agent
string json_routes 	# {vehicle_id : [[n11,n12,n13,...], [n21,n22,n23,...], ...]}
float64 total_quality	# total quality
float64 total_cost	# total cost
string json_capacity	# {vehicle_id : capacity}
int32[] unserved_customers	# a list of unserved customer indices
int32[] all_customers
string[] all_task_labels
string json_base_prec
string json_all_prec
string json_prec_matrix
